R/utils-dat.R

```diff
@@ -62,10 +64,48 @@ dat_install <- function (os = 'win', path, ver = '13.10.0') {
   unzip(normalizedDest, exdir = normalizedPath)
   file.remove(normalizedDest)
 
+  message("Modifying system PATH to enable easier access to 'dat'...")
+
+  # These still use `system()` vs `system2()`
```

Comment:
Any particular reason? If we're updating anyway then it makes sense to do it now? :100: